handbook/engineering/roles.md

```diff
@@ -8,13 +8,11 @@ Software engineers build our product and infrastructure.
 
 ### Responsibilities
 
-- Commit to a reasonable amount of work that you are going to get done for each [monthly release](releases/index.md), and then reliably get it done.
-- Post weekly updates about your progress on the [monthly release tracking issue](tracking_issues.md) for your team.
-- Ensure that your work is directly contributing to [our goals](../../company/goals/index.md).
-- Write [RFCs](../communication/rfcs/index.md) to recommend solutions to product and engineering problems.
-- Create and maintain high quality code, tests, and documentation.
-- Support your teammates by reviewing their [code](code_reviews.md) and [RFCs](../communication/rfcs/index.md).
-- Help us build a great team by referring people who you would like to work with again, interviewing candidates, and suggesting improvements to our hiring process.
+- Model [our company values](../../company/values.md) and apply our [guiding engineering principles](index.md#guiding-principles) throughout all your interactions and work.
+- Iteratively create and maintain high quality architecture, code, tests, and documentation that aligns with your team's goals.
```

Comment:
Do we want to talk about the "team's" goals, given that some engineers are not on teams?

Reply:
I think it is important to emphasize team goals given our value to work as a team. This still applies to people who are exceptions (currently there is only one exception): they should be helping teams achieve their goals. I would expect anyone in this "exception" situation to inherently know what their responsibilities are (otherwise they aren't the right person to be an exception). In the broader sense though, there are no exceptions, we are all on the same team: Sourcegraph.

Reply:
```suggestion
- Iteratively create and maintain high quality architecture, code, tests, and documentation that aligns with team goals.
```